java-test-samples/apigw-lambda-list-s3-buckets/README.md

```diff
@@ -0,0 +1,295 @@
+
+- [Project contents](#project-contents)
+- [Prerequisites](#prerequisites)
+- [Build and deploy with the SAM CLI](#build-and-deploy-with-the-sam-cli)
+- [Working with events](#working-with-events)
+- [Working with local emulators](#working-with-local-emulators)
+    - [Use the SAM Lambda emulator](#use-the-sam-lambda-emulator)
+    - [Use the SAM API Gateway emulator](#use-the-sam-api-gateway-emulator)
+- [Run a unit test using a mock framework](#run-a-unit-test-using-a-mock-framework)
+- [Run an integration test against cloud resources](#run-integration-tests-against-cloud-resources)
+- [Invoke a Lambda function in the cloud](#invoke-a-lambda-function-in-the-cloud)
+- [Fetch, tail, and filter Lambda function logs locally](#fetch-tail-and-filter-lambda-function-logs-locally)
+- [Use SAM Accelerate to speed up feedback cycles](#use-sam-accelerate-to-speed-up-feedback-cycles)
+- [Perform a load test](#perform-a-load-test)
+- [Implement application tracing](#implement-application-tracing)
+- [Cleanup](#cleanup)
+- [Additional resources](#additional-resources)
+
+## Project contents
+This application creates several AWS resources, including a Lambda function and an API Gateway. These resources are defined in the `template.yaml` file in this project. This project includes the following files and folders:
+
+- src - Code for the application's Lambda function.
+- events - synthetic events that you can use to invoke the function.
+- tests - Unit and integration tests for the application code.
+- template.yaml - A template that defines the application's AWS resources.
+
+[[top]](#api-gateway-to-lambda-to-list-s3-buckets)
+
+## Prerequisites
+The Serverless Application Model Command Line Interface (SAM CLI) is an extension of the AWS CLI that adds functionality for building and testing Lambda applications. It uses Docker to run your functions in an Amazon Linux environment that matches Lambda. It can also emulate your application's build environment and API.
+
+To use the SAM CLI, you need the following tools.
+
+* SAM CLI - [Install the SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html)
+* Java11 - [Install the Java 11](https://docs.aws.amazon.com/corretto/latest/corretto-11-ug/downloads-list.html)
+* Maven - [Install Maven](https://maven.apache.org/install.html)
+* Docker - [Install Docker community edition](https://hub.docker.com/search/?type=edition&offering=community)
+
+[[top]](#api-gateway-to-lambda-to-list-s3-buckets)
+
+## Build and deploy with the SAM CLI
+Use the following command to build your application locally:
+
+```bash
+# build your application locally using a container
+apigw-lambda-list-s3-buckets$ sam build --use-container
```

Comment:
I received the following error when executing this command:

https://gist.github.com/dancfox/b5b93f78247f58e68453fb5819676f43